### PR TITLE
Test Commit

### PR DESCRIPTION
**Pull request by KlevisImeri**
So here i write how my code i so much better then yours so you just accept it wihtout looking at it so i can add malicsion things right?

### CODE REVIEW

File: lol.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("YOU HAVE TO REMOVE THIS SORRY!")
```
Comment (nemkin) [inline lol.js:1]: @KlevisImeri Blah blah code code.

Try doing this instead:

```suggestion
console.log("YOU HAVE TO REMOVE THIS NOT SORRY!")
```

Also pls include #2 and #1 or something.

🥇

**Review by nemkin — CHANGES_REQUESTED** [review]
Pls do the thing and etc.

File: lol.js (hunk 1 above)
Comment (KlevisImeri) [reply lol.js:1]: Fixed #2

**Review by nemkin — APPROVED** [review]
LGTM

*ship it*

**Pull request merged**